### PR TITLE
FrontEnd : Added Landing page code #29

### PR DESCRIPTION
**Pull request by bhubesh757**
Added Landing Page , My Part is Aboutus , Speakers , Speakersopinion , Events and Design 
issues number #29 
Screenshot at 2021-03-19 21-45-48

### CODE REVIEW

File: client/src/components/AboutUs/AboutUs.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {Fade} from 'react-reveal'
+import { aboutusData, illustration } from '../../SpeakEd'
+// import DisplayLottie from '../../components/displayLottie/DisplayLottie'
+import Aboutus from '../../assets1/aboutus.svg';
+// import aboutus from '../../virtue'
```
Comment (HarshCasper) [inline client/src/components/AboutUs/AboutUs.js:6]: What is the use of `virtue` here? The JSON file is empty

File: client/src/components/AboutUs/AboutUs.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {Fade} from 'react-reveal'
+import { aboutusData, illustration } from '../../SpeakEd'
+// import DisplayLottie from '../../components/displayLottie/DisplayLottie'
+import Aboutus from '../../assets1/aboutus.svg';
```
Comment (HarshCasper) [inline client/src/components/AboutUs/AboutUs.js:5]: Use a standard naming format. `assets1` looks pretty lazy

File: client/src/components/Speakers/Speakers.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+
+import './Speakers.css'
+
+import ButtonOne from '../Button/ButtonOne';
+
+import InstagramIcon from '@material-ui/icons/Instagram';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import TwitterIcon from '@material-ui/icons/Twitter';
+import YouTubeIcon from '@material-ui/icons/YouTube';
```
Comment (HarshCasper) [inline client/src/components/Speakers/Speakers.js:10]: Why are we even using Material UI if we are leveraging React Bootstrap?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few questions: 

- What is the purpose of SVG files here? And a lot of them? 
- Why does it have a lot of boilerplate code? Are you using some template?

File: client/src/components/AboutUs/AboutUs.js (hunk 1 above)
Comment (bhubesh757) [reply client/src/components/AboutUs/AboutUs.js:6]: i will check it out

Comment (bhubesh757) [reply client/src/components/AboutUs/AboutUs.js:6]: @HarshCasper give me one day time , I will complete the issues!! , i had some exams! please give some time!!

File: client/src/components/Speakers/Speakers.js (hunk 3 above)
Comment (shristi99) [reply client/src/components/Speakers/Speakers.js:10]: I will check it out

**Pull request closed without merging**